server/resolvers/article/queries.js

```diff
@@ -32,6 +32,7 @@ const articleQueries = {
       delete article.wordCount
 
       cache.set(article.id, JSON.stringify(article))
+      cache.expire(article.id, 60 * 60 * 24)
```

Comment:
This method doesn't exist for the in-memory cache, right? What happens when you try to run this code with an in-memory cache?

Reply:
True lol.

I can go ahead and simulate that behavior with a timeout for the local storage.
Something like:
```
localCache.expire = (key, ttl) => setTimeout(() => localCache.delete(key), ttl * 1000)
```